### PR TITLE
[TOML] Move to encoding

### PR DESCRIPTION
**Pull request by zekth**
as mentionned in : https://github.com/denoland/deno_std/issues/431
Move toml to encoding.
Also added a cleanup using fs/EOL

### CODE REVIEW

**Comment by zekth** [comment]
Regarding comment in : https://github.com/denoland/deno_std/pull/432

Adjusted the path

File: encoding/toml.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { deepAssign } from "../util/deep_assign.ts";
 import { pad } from "../strings/pad.ts";
+import { format, EOL } from "../fs/eol.ts";
```
Comment (ry) [inline encoding/toml.ts]: This seems unnecessary https://deno.land/style_guide.html#withindeno_stdminimizedependenciesdonotmakecircularimports

File: encoding/toml.ts [hunk 2]
```diff
@@ -528,11 +529,11 @@ class Dumper {
...
   let out: string[] = [];
   out = new Dumper(srcObj).dump();
-  return out.join("\n");
+  return out.join(EOL.LF);
 }
 
 export function parse(tomlString: string): object {
   // File is potentially using EOL CRLF
-  tomlString = tomlString.replace(/\r\n/g, "\n").replace(/\\\n/g, "\n");
+  tomlString = format(tomlString, EOL.LF).replace(/\\\n/g, EOL.LF);
```
Comment (ry) [inline encoding/toml.ts]: I would just remove this line entirely. The parser should be able to handle either line ending.

Comment (zekth) [reply encoding/toml.ts]: If it's not here the parser will do it in itself. Example in golang csv reader: https://github.com/golang/go/blob/master/src/encoding/csv/reader.go#L236

File: encoding/toml.ts (hunk 1 above)
Comment (zekth) [reply encoding/toml.ts]: edited.

**Pull request merged**